m4_sessions/1708-inning/group_five.md

```diff
@@ -1,9 +1,21 @@
 # Group Five
 
-### Session Title:
+### Session Title: The Robots are Coming: The future of AI
 
 ### Session Outcomes: 
+- What does it mean to me as a programmer
+- Want people to consider how AI benefits or threatens their lives as devs
+- What AI is currently available, near future, and distant future
+- Understanding of what AI is (definitions and relevant vocabulary)
 
 ### Session Outline:
+- 10 to 15 min Intro (Icebreaker)
+- 20 min content video
```

Comment:
Think about narrowing down a video that his on one of your session outcomes.  Or potentially think about a video that presents more than one side of the conversation.